elixir/apps/web/lib/web/controllers/auth_controller.ex

```diff
@@ -161,7 +161,7 @@ defmodule Web.AuthController do
     )
     |> Web.Mailer.deliver_with_rate_limit(
       rate_limit_key: {:sign_in_link, identity.id},
-      rate_limit: 10,
+      rate_limit: 3,
```

Comment:
3 should be enough tokens, considering that they can use the most recent one even if they unsuccessfully tried to request a new one, right?

Reply:
3 seems reasonable to me